CHANGES

```diff
@@ -1,6 +1,13 @@
 3.2    unreleased
 =================
 
+Notable User Facing Changes
+---------------------------
+
+- Support for Clang/LLVM 16.0
```

Comment:
Did we add LLVM 16 support already?

Reply:
There are some bits in `cmake/LLVM.cmake`, but only the basic "version recognition" support.